orbit_predictor/predictors/tle.py

```diff
@@ -1,4 +1,18 @@
+import datetime as dt
+
 from .accurate import HighAccuracyTLEPredictor
+from ..utils import reify
 
 # Backwards compatibility
 TLEPredictor = HighAccuracyTLEPredictor
+
+
+class PersistedTLEPredictor(TLEPredictor):
+    def __init__(self, sate_id, source):
+        self._sate_id = sate_id
+        self._source = source
+        self._tle = self._source.get_tle(self.sate_id, dt.datetime.utcnow())
+
+    @reify
```

Comment:
If `self._source.get_tle` is called in the init, there is no need of the `@reify`, it is superfluos, since it is cacheing in `tle` the value that is already defined in `._tle` (was set in init time). Why not just set
 
`self.tle = self._source.get_tle(self.sate_id, dt.datetime.utcnow())`

in the init, and that's it.

Reply:
Fair enough!